dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/BeanDeploymentArchiveImpl.java

```diff
@@ -786,6 +797,9 @@ public void addEjbDescriptor(EjbDescriptor<?> ejbDescriptor) {
     public void addManagedBeanDescriptor(ManagedBeanDescriptor<?> managedBeanDescriptor) {
         if (getBeanDiscoveryMode() != BeanDiscoveryMode.NONE) {
             this.managedBeanClasses.add(managedBeanDescriptor.getBeanClass());
+            if (managedBeanDescriptor.getBeanClass() -= null) { 
```

Comment:
`-=` is probably not what you meant here

Reply:
Correct, that's a typo